File: src/SOP8/Egovframework/Base/base-model/src/main/java/egovframework/base/model/sop/component/TransmissionRegular.java
package egovframework.base.model.sop.component;

import lombok.*;
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class TransmissionRegular {
	public enum Fields { compn_sn, rgl_sn }

	private int compn_sn;

	private int rgl_sn;

	public static final String autoIncreaseField = null;
	public static final String tableName = "so_compn_trnsmis_rgl";
	public static final String[] primaryKeys = { "compn_sn", "rgl_sn" };

}

File: src/SOP8/Base/Base.Model/Sop/Component/TransmissionRegular.cs
using System;
using dnsDapperDBUtil.DataAccessLayer.IDAL;

namespace Base.Model.Sop.Component
{
	public class TransmissionRegular : Table
	{
		public enum Fields { compn_sn, rgl_sn };
		public enum WriteFields { compn_sn, rgl_sn };

		public int compn_sn { get; set; }
		public int rgl_sn { get; set; }

		public static string TableName { get { return "so_compn_trnsmis_rgl"; } }

		public override string GetTableName()
		{
			return TableName;
		}

		public override string GetPrimaryCondition()
		{
			return string.Format("{0} = {1} and {2} = {3}", Fields.compn_sn, compn_sn, Fields.rgl_sn, rgl_sn);
		}

		public override Type GetFieldType()
		{
			return typeof(Fields);
		}

		public override Type GetWriteFieldType()
		{
			return typeof(WriteFields);
		}

		public void FromCopy(TransmissionRegular obj)
		{
			this.compn_sn = obj.compn_sn;
			this.rgl_sn = obj.rgl_sn;
		}
	}
}
